server/controllers/services/email.js

```diff
@@ -92,7 +93,25 @@ export const messageSupport = async (req, res) => {
     },
   });
 
-  await emailLib.sendMessage(recipient, topic, html, options);
+  try {
+    await emailLib.sendMessage(recipient, topic, html, options);
+  } catch (e) {
+    reportErrorToSentry(e, {
+      user: remoteUser,
+      extra: {
+        body,
+        recipient,
+        topic,
+        html,
+        options,
+        ip,
+      },
+    });
+
+    res.status(500).send({
+      message: 'Error while opening the support ticket, please reach out directly to support@opencollective.com',
+    });
+  }
 
   res.status(200).send({ sent: true });
```

Comment:
We need to move this into the try scope because we can't `res.status.send` twice.

Reply:
Good catch, I forgot the `return`